### PR TITLE
updated dodge the ball minigame to api 8

### PR DESCRIPTION
**Pull request by SenjuZoro**
Updated dodge_the_ball minigame to api 8
And should be functional!

### CODE REVIEW

File: plugins/minigames/dodge_the_ball.py [hunk 1]
```diff
@@ -325,33 +327,34 @@ def check_ball_type(self, ball_type: BallType) -> NoReturn:
...
+            "mesh_scale", {
                 0.00: 1.4,
                 0.05: 1.7,
                 0.10: 1.4,
             }
         )
         # playing shoot sound.
-        ba.playsound(self.shoot_sound, position=self.node.position)
+        # self.shoot_sound, position = self.node.position.play();
+        self.shoot_sound.play()
```
Comment (rikkolovescats) [inline plugins/minigames/dodge_the_ball.py:352]: this won't take node's position into account.

i think `self.shoot_sound.play(position=self.node.position)` should fix it, will have to test tho.

Comment (SenjuZoro) [reply plugins/minigames/dodge_the_ball.py:352]: Surprisingly it worked!

**Comment by rikkolovescats** [comment]
we'll also need to create a `null` version meta entry for our updated dodge_the_ball minigame, check out https://github.com/bombsquad-community/plugin-manager/pull/164/commits/e4a317c8a93520619c0383f36876e91940294456 for an example

File: plugins/minigames.json [hunk 2]
```diff
@@ -457,6 +457,12 @@
         }
       ],
       "versions": {
+        "2.0.0": {
+          "api_version": 7,
+          "commit_sha": "88841cd",
+          "released_on": "23-07-2023",
+          "md5sum": "afca5167b564a7f7c0bdc53c3e52d198"
+        },
```
Comment (rikkolovescats) [inline plugins/minigames.json:465]: uh oh, seems like we updated the collector minigame's version meta, we'll have to remove this entire `2.0.0` object to revert the change, and create `null` meta entry for 2.0.0 version for dodge the ball minigame :cat:

**Review by rikkolovescats — APPROVED** [review]
yay ci's gone green tick, tysm!!

File: plugins/minigames.json (hunk 2 above)
Comment (SenjuZoro) [reply plugins/minigames.json]: check now 😭
